code/drasil-example/dblpend/lib/Drasil/DblPend/Requirements.hs

```diff
@@ -49,9 +50,7 @@ correct :: ConceptInstance
 correct = cic "correct" (foldlSent [
  atStartNP' (output_ `the_ofThePS` code), S "have the", 
  namedRef (propCorSol [] []) (plural propOfCorSol)]
- ) "Correct" nonFuncReqDom
+ ) "Correctness" nonFuncReqDom
 
 portable :: ConceptInstance
```

Comment:
This change makes it clear that several requirements (like `portable`) are repeated. As a subsequent PR, these should be pulled out of the example into `drasil-data`.